taskboot/cli.py

```diff
@@ -84,7 +91,7 @@ def main():
         '--build-arg',
         type=str,
         action='append',
-        help='Docker build args passed for each builded service',
+        help='Docker build args passed for each built service',
```

Comment:
I'm ashamed :man_facepalming:

Reply:
It happens to all of us not native speakers. :)